machine.go

```diff
@@ -118,6 +118,11 @@ if ! busybox test -L /bin ; then
 fi
 exec /lib/systemd/systemd
 `
+const udevRules = `
+# expose virtio-blk devices under /dev/disk/by-fakemachine-id
+KERNEL=="vd*[!0-9]", ATTRS{serial}=="?*", ENV{ID_SERIAL}="$attr{serial}", SYMLINK+="disk/by-fakemachine-id/$env{ID_SERIAL}"
+KERNEL=="vd*[0-9]", ATTRS{serial}=="?*", ENV{ID_SERIAL}="$attr{serial}", SYMLINK+="disk/by-fakemachine-id/$env{ID_SERIAL}-part%n"
```

Comment:
hmm this hardcodes that every backend will expose labels through the serial id which is rather limiting; actually as we're introducing udev rules we can also "remove" the restrictions on the label argument to be passed through..

e.g. one could imagine some backends simply would hardcode:
```
KERNEL=="vda", SYMLINK += "disk/by-fakemachine-id/badger"
KERNEL=="vda[0-9] SYMLINK+="disk/by-fakemachine-id/badger-part%n"
```

Reply:
ooh that's a good idea; right now the patch allows the backend to choose a udev match string. So I will make it so the backend can create a complete udev rule to be inserted into the fakemachine

Reply:
Right the only thing that should stay the probably stay consistant among backends is the SYMLINKS to be added. (e.g. disk/by-fakemachine-label/xxxx) for all of them and adjust the documentation for the label function a bit :p As currently it refers to exposing it as the SERIAL (which we can't guarantee), however the intention of this was to just make it possible to determine *which* block device matches what label.